data/ch/pricing.yml

```diff
@@ -0,0 +1,64 @@
+##################################### pricing ##############################
+pricing:
+  enable : true
+  title : Our Plans
+  pricing_table :
+    # pricing table loop
+    - name : Free
+      price : $99
+      unit : month
+      services:
+        - 1GB Disk Space
+        - 10 Email Account
+        - Script Installer
+        - 1 GB Storage
+        - 10 GB Bandwidth
+        - 24/7 Tech Support
+      button:
+        label : "Signup"
```

Comment:
We can remove this file (we'll add Chinese-language in a future PR using the go-18n for Hugo approach)